pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -225,6 +228,9 @@ func managePod(client coreclientv1.ConfigMapsGetter, recorder events.Recorder, o
 			required.Spec.InitContainers[0].Image = imagePullSpec
 		}
 	}
+	if len(operatorImagePullSpec) > 0 {
```

Comment:
do we ever have case when this is not set?

Reply:
> do we ever have case when this is not set?

improper wiring